### PR TITLE
added pagination support to /api/sources

### PR DESCRIPTION
**Pull request by attwad**

### CODE REVIEW

File: verification/curator-service/api/src/controllers/sources.ts [hunk 1]
```diff
@@ -3,10 +3,36 @@ import { Source } from '../model/source';
...
+    if (page < 1) {
+        res.status(422).json('page must be > 0');
+        return;
+    }
+    if (limit < 1) {
+        res.status(422).json('limit must be > 0');
+        return;
+    }
+    const docs = await Source.find({})
+        .skip(limit * page - limit)
```
Comment (axmb) [inline verification/curator-service/api/src/controllers/sources.ts:22]: hyper-nit: limit * (page - 1) is probably clearer

Comment (axmb) [inline verification/curator-service/api/src/controllers/sources.ts:22]: hyper-nit: limit * (page - 1) is slightly clearer IMO

Comment (attwad) [reply verification/curator-service/api/src/controllers/sources.ts:22]: done

**Comment by attwad** [comment]
let's see how we can generalize that, but it's quite simple and will only affect /list endpoints so let's see if we really need to do it.

**Pull request merged**